conda.recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
     - pip
     - cython
     - numpy
-    - setuptools >=3.3
+    - setuptools >=38.6.0
```

Comment:
this file is not used in the release process. I'm not sure if this has been maintained. Maybe worth removing.

Reply:
this is needed for the conda recipe, which is itself not used, so yeah can prob remove (in its entirely), but can do later